internal/output/templates.go

```diff
@@ -344,13 +344,16 @@ var CommentMarkdownWithHTMLTemplate = `
 </table>
 {{- end }}
 
+{{- if not .MarkdownOptions.OmitDetails }}
+
```

Comment:
Adding this to html+markdown template to keep all templates similar to each other as much as possible.